packages/grafana-data/src/field/fieldOverrides.test.ts

```diff
@@ -802,8 +802,8 @@ describe('getLinksSupplier', () => {
 
     const links = supplier({ valueRowIndex: 0 });
     const rangeStr = JSON.stringify({
-      from: range.from.toISOString(),
-      to: range.to.toISOString(),
```

Comment:
why is this needed?

Reply:
This makes internal URLs generated by data links consistent with Explore URLs (uses ms instead of ISO)